File: adwwdas/adwwdas/Rentura - MovieTime.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Project/Maven2/JavaApp/src/main/java/${packagePath}/${mainClassName}.java to edit this template
 */

package com.mycompany.movietime;

/**
 *
 * @author COMLAB101
 */

import java.util.LinkedList;
import java.util.Queue;
import java.util.Scanner;

public class MovieTime {

    public static void main(String[] args) {
        
    String movieQue, snackQue;
    
    Queue <String> movies = new LinkedList <>();
    Queue <String> snacks  = new LinkedList <> ();
    
    Scanner sc = new Scanner (System.in);
   
     for(int i = 1; i <= 3 ; i++){
         System.out.println( "Enter movie " + i + " of 3: ");
         movieQue = sc.nextLine();
         movies.offer(movieQue);
     }
     
     
     for(int i = 1; i <= 3 ; i++){
         System.out.println( "Enter snack" + i + " of 3: ");
         snackQue = sc.nextLine();
         snacks.offer(snackQue);
     }
     
        System.out.println("Movies to watch are: " + movies);
        System.out.println("Snacks available are: " + snacks);
        System.out.println("Press 'S' each time you finish a snack.");
     
     while(!snacks.isEmpty()){
         
        String key = sc.nextLine();
            
        if (key.equalsIgnoreCase("S")) {
           snacks.poll();
           System.out.println(snacks);
           
     }
        
     }

        if (snacks.isEmpty()) {
            System.out.println("No more snacks");
        }
    }
}

File: adwwdas/adwwdas/Rentura - MovieTime.py
from queue import Queue

movies = Queue()
snacks = Queue()

for i in range(1, 4):
    movieQue = input("Enter movie " + str(i) + " of 3: ")
    movies.put(movieQue)

for i in range(1, 4):
    snackQue = input("Enter snack " + str(i) + " of 3: ")
    snacks.put(snackQue)

print("Movies to watch are: " + str(list(movies.queue)))
print("Snacks available are: " + str(list(snacks.queue)))
print("Press 'S' each time you finish a snack.")

while not snacks.empty():
    key = input()
    if key.upper() == "S":
        snacks.get()
        print(list(snacks.queue))

if snacks.empty():
    print("No more snacks")